src/config/footer.js

```diff
@@ -4,11 +4,14 @@ import { hostnameContext } from '../helper/url';
 const FooterData = {
   info: [
     {
-      name: 'Docs',
-      url: 'https://docs.openlaw.io',
-    }, {
       name: 'About',
       url: '/about?ref=footer',
+    }, {
+      name: 'Github',
```

Comment:
I might be nitpicking too hard here, but I think "Github" --> "GitHub" :-)

Reply:
Haha, sure thing! This is like ConsenSys vs Consensys ;)